### ISSUE TITLE
Time for a parse-dashboard Guide?

### ISSUE THREAD

**Issue by OpConTech** [parse-community/parse-dashboard#1653]
Make sure these boxes are checked before submitting your issue -- thanks for reporting issues back to Parse Dashboard!

- [X] You're running version >=1.0.23 of Parse Dashboard.

- [X] You're running version >=2.3.2 of Parse Server.

- [X] You've searched through [existing issues](https://github.com/ParsePlatform/parse-dashboard/issues?utf8=%E2%9C%93&q=). Chances are that your issue has been reported or resolved before.

So...just trying to be helpful make CLP's more intuitive. I know you all work very hard.
Here is some feedback I hope is helpful that is related to a parse-server issue
(ref issue - https://github.com/parse-community/parse-server/issues/7175 ):

parse-dashboard feedback: Had some trouble when clicking on CLP > gear icon. Once it started working it worked every time. It's like some how in my browser it was stuck in the greyed out state and would not highlight black when I read the mouse over it.
Maybe replace the gear icon with just the Simple-Advance switch...Or maybe just get rid of all of that and show all the CLPs when the user clicks Security (keep it simple...less to code...less to test...less to document.)

Documentation feedback: Maybe its time to have a parse-dashboard Guide. Parse docs have come a long way since Facebook gave Parse to the community except for parse-dashboard. There have been a ton of great new features like GraphQL console that  you all have built since 2017. And there are some issues in other parse guides that really would be better documented or co-documented in a parse-dashboard guide - e.g. Analytics appears in the iOS Guide https://docs.parseplatform.org/ios/guide/#analytics but Analytics does not exist in parse-dashboard any more.

Hope this feedback is helpful and thanks for all that you do for parse-dashboard.

**Comment by mtrezza** [comment]
Thanks for suggesting.

This has been discussed recently and due to the increasing complexity of Parse Dashboard this could help to onboard new users as well as help existing users to get familiar with new or more complex features.

Maybe this could also be an opportunity to try out a new docs framework.

**Comment by OpConTech** [comment]
Exactly. Going further with making complex features easier to understand by adding a parse-dashboard guide is the gist of my feedback.

I knew from the "permission denied" error code that I had an issue. But was it ACL, CLP or Protected Fields?  So I jumped into parse-dashboard and found the new Security tab which was great. But even with that new feature I was unable to resolve my issue. Once I saw @dplewis posted screen shot of Advanced CLP in parse-dashboard I quickly solved my issue.  

I can draft up a TOC that is based on the flow and format of the parse-server and sdk guides.  

Also what is the new docs framework, Is Parse moving away from more Jekyll?

**Comment by mtrezza** [comment]
> Also what is the new docs framework, Is Parse moving away from more Jekyll?

Just a suggestion because I see some shortcomings in the usability, layout and functionality of the current guides. Ideally there is only one guide for all SDKs instead of separate docs per SDK with some common pages. Hence I'm suggesting if there is a better framework we want to transition to, this could be a starter.

**Comment by mtrezza** [comment]
Maybe we can take this opportunity and create the Parse Dashboard docs trying out https://docusaurus.io. It seems to be one of the most advanced docs frameworks, open source and seems to be backed by FB Open Source with a large community.

@TomWFox What do you think about this and how much effort would it be to set this up on our server?

**Comment by TomWFox** [comment]
I think that's a good idea. I played around with Docusaurus a while a go and it was pretty easy to setup and required fairly minimal changes to bring in the old docs.

**Comment by mtrezza** [comment]
Update: We added a Feature section to the README page to at least already start to collect content for a future dashboard guide.
